spec/serializers/api/v1/offer_serializer_spec.rb

```diff
@@ -10,5 +10,7 @@
     expect(json['offer']['id']).to eql(offer.id)
     expect(json['offer']['language']).to eql(offer.language)
     expect(json['offer']['state']).to eql(offer.state)
+    expect(json['offer']['submitted_at'].to_date).
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.